apps/www/_blog/2023-04-14-pg-tle.mdx

```diff
@@ -0,0 +1,66 @@
+---
+title: 'User-managed Trusted Language Extensions for Supabase and RDS'
+launchweek: 7
+tags:
+  - launch-week
+  - tle
+date: '2023-04-14'
+to_depth: 3
+author: michel,oli_rice
+image: launch-week-7/day-2-edge-runtime/edge-runtime-og.png
+thumb: launch-week-7/day-2-edge-runtime/edge-runtime-thumb.jpeg
+---
+
+# User-managed Trusted Language Extensions for Supabase and RDS
+
+## Summary
+
+Most PostgreSQL cloud providers, such as Supabase and AWS RDS, do not allow users to install Postgres extensions into their databases until they have been rigorously vetted. Because of this, many useful Postgres extensions cannot be used on cloud platforms. However, the RDS has developed a new kind of extension called a **Trusted Language Extension** has been introduced that allows extensions written in Trusted Languages to be created from any database user without requiring filesystem access.
```

Comment:
```suggestion
Most PostgreSQL cloud providers, such as Supabase and AWS RDS, do not allow users to install Postgres extensions into their databases until they have been rigorously vetted. Because of this, many useful Postgres extensions cannot be used on cloud platforms. To solve this, the RDS team has developed a new kind of extension called a **Trusted Language Extension** (TLE). 

A TLE is a Postgres extension that is written in a Trusted Language, and it can be installed in any Postgres provider that has the [pg_tle](https://github.com/aws/pg_tle) extension installed. The goal is to make more extensions available to developers, even if your database is hosted on the cloud.
```